ideascube/configuration/registry.py

```diff
@@ -12,19 +12,29 @@
 REGISTRY = {
     # 'namespace': {
     #     'key': {
+    #         'summary': 'Help text for the management command',
+    #         'pretty_type': 'Pretty type(s) for the management command',
     #         'type': str,
     #         'default': 'Default value',
     #     },
     # },
     'server': {
         'site-name': {
+            'summary': 'The pretty name of the server, as seen by users of '
+                       'the web interface.',
+            'pretty_type': 'A string',
             'type': str,
             'default': getattr(settings, 'IDEASCUBE_NAME', 'Ideas Cube'),
         },
     },
 }
 
 
+def get_all_namespaces():
+    for namespace in sorted(REGISTRY):
+        yield namespace
+
```

Comment:
You know what? This is the kind of syntactic sugar I normally enjoy, and the only thing I had against is lack of familiarity.

But I actually want to get familiar with this one. It's elegant, quick, and not entirely obscure.

So I changed it in the latest push to this branch.